torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -349,15 +349,15 @@ def ReplaceMoreTtManually(gm: torch.fx.GraphModule) -> torch.fx.GraphModule:
                 if args[0] >= 2:
                     # step = 1
                     new_args = (args[0],)
-                    new_kwargs = {"end": args[1], "step": 1, "device": TtnnDevice()}
+                    new_kwargs = {"stop": args[1], "step": 1, "device": TtnnDevice()}
```

Comment:
@kevinwuTT do we need to pass Device like this? 
I thought it is available as a compiler option right now, maybe even set globally as the default device

Reply:
It might not be needed anymore. @jdh8 Can you try without?

Reply:
I cleared the `device` argument and moved others to positional arguments.  It worked well in the op tests.  However, model tests kept failing in my environment, even after I checked out at the `main` branch and updated `tt-metal`.  We cannot locally produce a reliable regression test for now.